app/src/ui/app-menu/menu-list-item.tsx

```diff
@@ -161,6 +178,13 @@ export class MenuListItem extends React.Component<IMenuListItemProps, {}> {
       selected: this.props.selected,
     })
 
+    const role = this.props.hasNoRole
+      ? undefined
+      : type === 'checkbox'
```

Comment:
So.. this should probably be the radio one... but it uses the wrong icon. :/ May circle back and allow for a custom icon.